{{cookiecutter.project_name}}/cookiecutter-options.yml

```diff
@@ -0,0 +1,9 @@
+default_context:
+    project_name: "{{ cookiecutter.project_name }}"
+    app_name: "{{ cookiecutter.app_name }}"
+    python_version: "{{ cookiecutter.python_version }}"
+    admin_user_username: "admin"
+    admin_user_email: "admin@brightedge.com"
```

Comment:
set to `admin@mail.com` in `cookiecutter.json`

Reply:
haha. Whoops!!